texttospeech/synthesize_file/synthesize_file_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+const TEXT_FILE = "../resources/hello.txt"
+const SSML_FILE = "../resources/hello.ssml"
+const OUTPUT_FILE = "output.mp3"
```

Comment:
UPPER_CAPS is not the style for Go constants.

Just

```
const (
  textFile = "..."
)
```

Reply:
Got it, changed